templates/components/common/header.html

```diff
@@ -16,12 +16,17 @@
         <h1 class="header-logo header-logo--{{theme_settings.logo-position}}">
             {{> components/common/store-logo}}
         </h1>
+        
+        {{{region name="homepage-banner"}}}
     {{else}}
         <div class="header-logo header-logo--{{theme_settings.logo-position}}">
             {{> components/common/store-logo}}
         </div>
     {{/if}}
 
+    {{#if template_file '===' 'pages/cart'}}
```

Comment:
Please don't do this. Instead use the `partial` helper.

In `pages/cart.html` add
```
{{#partial "head"}}
  {{{region name="cart-page-banner"}}}
{{/partial}}
```